src/inmanta/protocol/methods.py

```diff
@@ -690,7 +690,7 @@ def set_param(
         :param tid: The id of the environment
         :param id: The name of the parameter
         :param resource_id: Optionally, scope the parameter to resource (fact)
-        :param source: The source of the parameter, this can be the user, agent, plugin, compiler, ...
+        :param source: The source of the parameter. Valid values are defined in the SOURCE list in inmanta/data/init.py
```

Comment:
perhaps this should be made into a proper enum?

Reply:
done